### PR TITLE
Improve Mendix Cloud V4 documents

### PR DESCRIPTION
**Pull request by MarkvanMents**
V4 documents should be more enticing - discussion with Tom dG

### CODE REVIEW

File: content/developerportal/deploy/mxcloudv4.md [hunk 1]
```diff
@@ -8,25 +8,28 @@ tags: ["Cloud", "Mendix Cloud", "V4", "Version 4", "FAQ", "V3", "AWS", "Amazon W
...
 
-V4 was launched in Q1 2017. As in v3, applications are deployed via the Mendix Developer Portal or our [APIs](/apidocs-mxsdk/apidocs/deploy-api). Unlike v3, the applications run on Cloud Foundry clusters that are deployed on highly available Amazon Web Services (AWS) regions. Apps can run in the EU, US, Japan, or the UK.
+Apps deployed to V4 run in Cloud Foundry clusters that are deployed on highly available Amazon Web Services (AWS) regions. They can be deployed from Studio Pro and the Mendix Developer Portal or via our [APIs](/apidocs-mxsdk/apidocs/deploy-api). Apps can run in the EU, US, Japan, or the UK.
 
-## 2 What Is the Status of Mendix Cloud v3?
+Mendix Cloud v4 is a mature product, launched in Q1 2017, and has a number of improvements over previous versions of Mendix Cloud:
 
-New apps will be launched on Mendix Cloud v4 by default. Customers that need to stay on v3 because they use a VPN can still get new apps on v3 for the foreseeable future. Mendix Cloud v3 will be supported for several more years; no end-of-support or end-of-life dates have been set yet.
+* It is more secure
```
[the remark this answers is not in the document]
Comment (MarkvanMents) [reply content/developerportal/deploy/mxcloudv4.md]: Added

**Comment by MarkvanMents** [comment]
@tomedegroot - I've made an improvement to the security description.
@feddovanede - Can you (or someone else in support) review this today so I can publish before I go on holiday. Thanks.

File: content/developerportal/deploy/migrating-to-v4.md [hunk 2]
```diff
@@ -14,33 +14,57 @@ This how-to explains how to migrate your app from a Mendix Cloud v3 node to a Me
...
-
 * Create a new temporary Free App without a sandbox environment – instructions for unlinking a Free App from a sandbox are here: [Licensing Mendix Cloud Apps](licensing-apps#unlink)
 
-## 3 Migrating the App
+## 3 Migration considerations
+
+You will need to take the following into account when planning to migrate to Mendix Cloud V4:
+
+* Your app needs to be on a supported version of Mendix, that is **Mendix version 6.0 or above**
+* Long-running scheduled events will probably need to be split into smaller chunks using a queueing system like the Amazon SQS connector to spread the work out over multiple instances
```
Comment (feddovanede) [inline content/developerportal/deploy/migrating-to-v4.md]: This only applies to Mendix 7, I think? I don't think you can horizontally scale Mendix 6 apps?

Also, shouldn't this be written in a more directing way: 'You might want to...'?

File: content/developerportal/deploy/migrating-to-v4.md [hunk 3]
```diff
@@ -14,33 +14,57 @@ This how-to explains how to migrate your app from a Mendix Cloud v3 node to a Me
...
+* If you use a mail server from your app, you will need to use a third-party email provider – for more information, see [Sending Email](sending-email)
+* To secure your app, you will have to use one of the recommended methods discussed in [Securing Outgoing Connections from Your Application](securing-outgoing-connections-from-your-application) if you are currently using a VPN.
```
Comment (feddovanede) [inline content/developerportal/deploy/migrating-to-v4.md]: Shouldn't it start with 'If you use a VPN connection...', like the previous sentence?

File: content/developerportal/deploy/migrating-to-v4.md [hunk 4]
```diff
@@ -14,33 +14,57 @@ This how-to explains how to migrate your app from a Mendix Cloud v3 node to a Me
...
+
+To make the most of the features of Mendix Cloud V4, we recommend that your apps are built as [12-factor apps](https://12factor.net/).
+
+## 3.1 Other Differences Between Mendix Cloud V3 & V4
+
+There are a few other differences between the way you develop and deploy apps in Mendix Cloud v4 and Mendix Cloud v3:
+
+* In Mendix Cloud v4, the debugger is always active, and does not have to be activated – the button shows the credentials to connect Mendix Studio Pro to the debugger
```
Comment (feddovanede) [inline content/developerportal/deploy/migrating-to-v4.md]: What button? Should we list the limitation of the debugger working only when you have 1 instance, here as well?

File: content/developerportal/deploy/migrating-to-v4.md [hunk 5]
```diff
@@ -64,37 +88,25 @@ To upload the deployment package, follow these steps:
...
-With the node removed, the app will still remain in the [Developer Portal](http://home.mendix.com) as a Free App.
-
-{{% alert type="warning" %}}
-
-Make sure you have downloaded the latest backup and deployment package before offboarding the node.
-
-{{% /alert %}}
-
-### 3.4 Linking the App to the v4 Node
+### 4.3 Linking the App to the v4 Node
```
Comment (feddovanede) [inline content/developerportal/deploy/migrating-to-v4.md]: Offboarding is only in step 4.6, _after_ this step?

File: content/developerportal/deploy/migrating-to-v4.md [hunk 6]
```diff
@@ -119,11 +131,23 @@ To change the App URL (if you are not using a custom domain) you will need to co
...
 
-### 3.6 Changing a Custom Domain
+### 4.5 Changing a Custom Domain
 
 If you have a custom domain set up, you will need to change this. Instructions for doing this are here: [Custom Domains](custom-domains).
 
-## 4 Read More
+### 4.6 Offboarding the v3 Node
+
+Your app is now running in Mendix Cloud v4. If everything works correctly, submit a request at [Mendix Support](https://support.mendix.com) to offboard the v3 node. This means that the app will be detached from the node.
```
Comment (feddovanede) [inline content/developerportal/deploy/migrating-to-v4.md]: You detach the app yourself from the v3 node in step 4.3?

File: content/developerportal/deploy/migrating-to-v4.md [hunk 7]
```diff
@@ -119,11 +131,23 @@ To change the App URL (if you are not using a custom domain) you will need to co
...
+
+With the node removed, the app will still remain in the [Developer Portal](http://home.mendix.com) as a Free App.
```
Comment (feddovanede) [inline content/developerportal/deploy/migrating-to-v4.md]: The app is linked to the v4 node in step 4.3?

File: content/developerportal/deploy/mxcloudv4.md [hunk 8]
```diff
@@ -8,25 +8,28 @@ tags: ["Cloud", "Mendix Cloud", "V4", "Version 4", "FAQ", "V3", "AWS", "Amazon W
...
+Apps deployed to V4 run in Cloud Foundry clusters that are deployed on highly available Amazon Web Services (AWS) regions. They can be deployed from Studio Pro and the Mendix Developer Portal or via our [APIs](/apidocs-mxsdk/apidocs/deploy-api). Apps can run in the EU, US, Japan, or the UK.
 
-## 2 What Is the Status of Mendix Cloud v3?
+Mendix Cloud v4 is a mature product, launched in Q1 2017, and has a number of improvements over previous versions of Mendix Cloud:
 
-New apps will be launched on Mendix Cloud v4 by default. Customers that need to stay on v3 because they use a VPN can still get new apps on v3 for the foreseeable future. Mendix Cloud v3 will be supported for several more years; no end-of-support or end-of-life dates have been set yet.
+* It is more secure as you can use [HTTP Headers](environments-details#http-headers) and [client certificates](certificates), and have [TLS v1.2 or higher enforced](/releasenotes/developer-portal/#january-28th-2019)
+* It runs on faster hardware
```
Comment (feddovanede) [inline content/developerportal/deploy/mxcloudv4.md]: Do we have an oversight of the hardware used?

File: content/developerportal/deploy/mxcloudv4.md [hunk 9]
```diff
@@ -8,25 +8,28 @@ tags: ["Cloud", "Mendix Cloud", "V4", "Version 4", "FAQ", "V3", "AWS", "Amazon W
...
+* It is still being actively developed and improved, with new features regularly being introduced (for example [flexible environments](mendix-cloud-deploy#flexible-environments))
+* It uses industry-standard techniques for scalability and maintainability
```
Comment (feddovanede) [inline content/developerportal/deploy/mxcloudv4.md]: What industry-standard techniques?

File: content/developerportal/deploy/mxcloudv4.md [hunk 10]
```diff
@@ -8,25 +8,28 @@ tags: ["Cloud", "Mendix Cloud", "V4", "Version 4", "FAQ", "V3", "AWS", "Amazon W
...
+* It supports all current versions of Mendix, including Mendix version 8
+* It has improved monitoring features, including using some external monitoring tools
```
Comment (feddovanede) [inline content/developerportal/deploy/mxcloudv4.md]: Examples?

File: content/developerportal/deploy/mxcloudv4.md [hunk 11]
```diff
@@ -46,63 +49,40 @@ Data will always be stored in the same political region for the following region
 
 Data in Japan is currently backed up in Australia.
```
Comment (feddovanede) [inline content/developerportal/deploy/mxcloudv4.md]: Shouldn't this be a bullet in the previous list?

File: content/developerportal/deploy/mxcloudv4.md [hunk 12]
```diff
@@ -46,63 +49,40 @@ Data will always be stored in the same political region for the following region
...
-* The number of permitted database connections is no longer static, but tied to the RAM of the database environment. The limit is roughly 100 connections per GB of database RAM. You can use the Mendix Runtime settings **ConnectionPoolingMaxActive** and **ConnectionPoolingMaxIdle** to tweak the number of database connections that the Mendix Runtime will set up. The defaults are perfectly fine for most situations. Keep in mind that the setting is applied per runtime instance, so the configured number is multiplied by the number of instances.
-
-## 10 Missing in Mendix Cloud v4
-
-There are some features missing in v4. Mendix will implement the following features in the future:
+You can also consider running your Mendix app using your own AWS account in a different AWS region. You can do this using Docker, and there is some information on how to do this in the [Docker](docker-deploy) documentation. If you do this, however, you will not receive all the benefits of running in the Mendix Cloud.
```
Comment (feddovanede) [inline content/developerportal/deploy/mxcloudv4.md]: 'Some information' doesn't really sound like we really support it?

File: content/developerportal/deploy/mxcloudv4.md [hunk 13]
```diff
@@ -46,63 +49,40 @@ Data will always be stored in the same political region for the following region
...
 
-* File storage usage is not visible
-* Application CPU alerts are not sent
-* Archived logs can only be downloaded, not viewed in the browser
-* The database status is not visible on the node details screen
+## 7 What Other Considerations Are There When Running My App in Mendix Cloud V4?
```
Comment (feddovanede) [inline content/developerportal/deploy/mxcloudv4.md]: Why isn't this listed in the other document?

**Comment by MarkvanMents** [comment]
Thanks for all your comments. I will now merge this.

**Pull request merged**